app/views/duties/index.html.erb

```diff
@@ -3,19 +3,22 @@
 
 <div class="container-fluid no-padding-top">
   <div class="row">
+    <div class="col-md-12 no-padding">  
+      <h1>Duty Timetable for the Week</h1>
+    </div>
+  </div>
+
+  <div class="row body-container">
     <div id="announcement-sidebar" class="col-md-3 sidebar-container">
       <h1>Announcements</h1>
-      <%= render partial: 'announcements/index' %>
+        <%= render :partial => 'announcements/index' %>
```

Comment:
Lol both are equivalent but I think Herbert prefers the `{ key_in_symbol: value }` syntax

Reply:
@guekling